sharding-core/sharding-core-api/src/main/java/org/apache/shardingsphere/api/config/sharding/ShardingRuleConfiguration.java

```diff
@@ -45,6 +45,8 @@
     private Collection<String> broadcastTables = new LinkedList<>();
     
     private String defaultDataSourceName;
+
+    private String timeServiceDataSourceName;
```

Comment:
Please do not modify sharding configuration. `timeServiceDataSourceName` should use properties from SPI